charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/mutatingWebhookConfiguration.yaml

```diff
@@ -41,4 +41,26 @@ webhooks:
     timeoutSeconds: {{ .Values.prometheusOperator.admissionWebhooks.timeoutSeconds }}
     admissionReviewVersions: ["v1", "v1beta1"]
     sideEffects: None
+    {{- if or .Values.prometheusOperator.denyNamespaces .Values.prometheusOperator.namespaces }}
+    namespaceSelector:
+      matchExpressions:
+      {{- if .Values.prometheusOperator.denyNamespaces }}
+      - key: kubernetes.io/metadata.name
+        operator: NotIn
+        values:
+        {{- range $namespace := .Values.prometheusOperator.denyNamespaces }} 
+        - {{ $namespace }}
+        {{- end }}
+      {{- else if and .Values.prometheusOperator.namespaces .Values.prometheusOperator.namespaces.additional }}
+      - key: kubernetes.io/metadata.name
+        operator: In
+        values:
+        {{- if and .Values.prometheusOperator.namespaces.releaseNamespace (default .Values.prometheusOperator.namespaces.releaseNamespace true) }}
```

Comment:
I would use the same syntax as the one used here https://github.com/prometheus-community/helm-charts/blob/22b98bae93a67016082ec883d4d846c1282b5bf7/charts/kube-prometheus-stack/templates/prometheus-operator/deployment.yaml#L75

Also it is best to use https://github.com/prometheus-community/helm-charts/blob/22b98bae93a67016082ec883d4d846c1282b5bf7/charts/kube-prometheus-stack/templates/prometheus-operator/deployment.yaml#L1 for the release.namespace for cases when the operator is used as a subchart.

Could you address those and bump the version?

Reply:
@QuentinBisson  Yes, working on it right now

Reply:
@QuentinBisson I don't really understand how the line 58 can be transform in order to use syntax from the line 75 in deployment.yaml. Can you elaborate on it a little bit more, please? 

Regarding your second comment, am I understanding it right, that it is related to the line 59, and you want me to change `- {{ .Release.Namespace }}` to something like `{{- $namespace := printf "%s" (include "kube-prometheus-stack.release.namespace" .) }} ` ?

Reply:
Yes you are right for the second comment, I think you added a line but the comment stayed on this line number for some reason 

For the second part, I would maybe just use mustUnique everywhere and that would be fine

Reply:
@QuentinBisson I addressed an issue regarding `release.namespace`. However, I still don't understand how `mustUnique` is related to the if statement on the line 58. Maybe you want me to exclude any potentially similar namespaces in `.Values.prometheusOperator.namespaces.additional` ?

Reply:
Yes I would avoid duplicates in either denyNamespaces or .Values.prometheusOperator.namespaces.additional :)

Reply:
@QuentinBisson  I addressed issues you mentioned. Can you take a look again, please?